### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by kaliomar** [psf/requests#5315]
When i import requests and run help(requests), Output is this:

> ESCRIPTION
    Requests HTTP Library
    ~~~~~~~~~~~~~~~~~~~~~
    
    Requests is an HTTP library, written in Python, for human beings. Basic GET
    usage:
    
       >>> import requests
       >>> r = requests.get('https://www.python.org')
       >>> r.status_code
       200
       >>> 'Python is a programming language' in r.content
       True

But when i try this 'Python is a programming language' in r.content, Output doesn't be True, It says

> TypeError: a bytes-like object is required, not 'str'

**Comment by Recursing** [comment]
This should be already fixed in master https://github.com/psf/requests/blob/d2590ee46c0641958b6d4792a206bd5171cb247d/requests/__init__.py#L19

**Comment by nicktimko** [comment]
`'Python is a programming language' in r.text` is also an alternative (which maybe works in both 2.7 and 3.x?).

**Comment by Recursing** [comment]
The version in master also works both in 2.7 and 3.x

**Comment by nateprewitt** [comment]
@kaliomar, Recursing is correct. This was fixed in #4987 in August. The error you're hitting is the example is trying to compare content (a byte string) with a unicode string in Python 3 which won't work. This example worked in Python2 since both strings were byte-strings.

It looks like this wasn't merged until after the last release of Requests, but will be corrected in the next. Thanks for bringing it to our attention.
